Mathlib/RingTheory/Polynomial/Quotient.lean

```diff
@@ -70,6 +70,14 @@ noncomputable def quotientSpanCXSubCXSubCAlgEquiv {x : R} {y : R[X]} :
       (Ideal.Quotient.algebra R) _ :=
 ((quotientSpanCXSubCAlgEquiv (X - C x) y).restrictScalars R).trans <| quotientSpanXSubCAlgEquiv x
 
+lemma modByMonic_eq_zero_iff_quotient_eq_zero (p q : R[X]) (hq : q.Monic) :
+    p %ₘ q = 0 ↔ (p : R[X] ⧸ Ideal.span {q}) = 0 := by
+  rw [dvd_iff_modByMonic_eq_zero hq, Ideal.Quotient.eq_zero_iff_dvd]
```

Comment:
Is this lemma really worth having? It's a straightforward combination of simpler lemmas

Reply:
It was not immediately obvious to me how to prove it, I didn't know whether it would go through `dvd` or `mod` or what. I figure it won't be hard to maintain and it might solve someone's docs search a bit faster at some point.